imports/api/Testing/createNewExperiences.js

```diff
@@ -224,10 +223,7 @@ function createNewStorytime() {
             options: [
               ["Drinking butterbeer", "N3uajhH3chDssFq3r"],
               ["Hogwarts Express at Platform 9 3/4", "Ly9vMvepymC4QNJqA"],
-              [
-                "Sneaking around at night under the invisibility cloak",
-                "AcstpXRyNFhmgPDfF"
-              ],
+              ["Sneaking around at night under the invisibility cloak", "AcstpXRyNFhmgPDfF"],
```

Comment:
For this entire file, we are doing two things wrong

1. Some experience definitions are duplicates to testingconstants.  In those places, we should do something like
```
  let exp = CONSTANTS.EXPERIENCES.natureHunt;
```

2. This is a better idea, since the old definitions in this file do not have the notificationDelay parameter

Reply:
I'm good with that, but are the experiences here the same as the `testingconstants.js` file? If so, go for it. Otherwise, we may want to alter the experiences in `testingconstants.js` for alignment and then import them in